hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableAccountStore.java

```diff
@@ -122,6 +125,14 @@ public void removeAlias(@NonNull final Bytes alias) {
         // We really shouldn't ever see an empty alias. But, if we do, we don't want to do any additional work.
         // FUTURE: It might be worth adding a log statement here if we see an empty alias, but maybe not.
         if (alias.length() > 0) {
+            if (!isOfEvmAddressSize(alias)) {
+                final var key = asKeyFromAlias(alias);
+                final var evmAddress = extractEvmAddress(key);
+                if (evmAddress != null) {
+                    aliases().remove(new ProtoBytes(evmAddress));
+                    entityCounters.decrementEntityTypeCounter(EntityType.ALIAS);
```

Comment:
Seemed not correct that we are decrementing twice here. But seems like we are incrementing twice in `CryptoCreateHandler`. So, this is correct.